dataland-frontend/src/components/resources/dataTable/MultiLayerDataTableCell.vue

```diff
@@ -1,16 +1,16 @@
 <template>
-  <div class="flex" v-if="content.displayComponent == MLDTDisplayComponents.HighlightHiddenCellDisplayComponent">
+  <div class="flex" v-if="content.displayComponentName == MLDTDisplayComponents.HighlightHiddenCellDisplayComponent">
     <i class="pi pi-eye-slash pr-1 text-red-500" aria-hidden="true" data-test="hidden-icon" />
     <MultiLayerDataTableCell :content="content.displayValue.innerContents" />
   </div>
-  <component v-else :is="content.displayComponent" :content="content" />
+  <component v-else :is="content.displayComponentName" :content="content" />
 </template>
 
 <script lang="ts">
 import {
-  type AvailableDisplayValues,
-  MLDTDisplayComponents,
-} from "@/components/resources/dataTable/MultiLayerDataTableCells";
+  type AvailableMLDTDisplayObjectTypes,
+  MLDTDisplayComponentName,
+} from "@/components/resources/dataTable/MultiLayerDataTableCellDisplayer";
```

Comment:
That was just a renaming of the file, correct?

Reply:
Yes. I wanted to help other people understanding the MLDT better.
That's why I renamed everything that confused me during my time of digging into the MLDT. 
I hope it helps a little bit.

Reply:
ok, i will resolve here